setup.py

```diff
@@ -31,6 +31,7 @@
     zip_safe=False,
     python_requires=">3, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4",
     install_requires=[
+        "chardet>=4.0.0",
```

Comment:
is this needed for install or only to run tests?

If actually for install, we should get this released right away since fresh installs won't work.
If only needed for tests, we can move chardet to dev dependencies.

Reply:
For install. We're using it to detect file encodings when we read bulk files.

Reply:
sounds good, is master in a state where we can safely bump the version and cut a release? Might as well bump the version in this PR too if that is the case.

Reply:
I bumped the version and made a changelog note so we could release on this PR.